docs/book/src/migration/v1vsv2.md

```diff
@@ -0,0 +1,59 @@
+# Kubebuilder v1 vs v2
+
+This document should cover all breaking changes when migrating from v1 to v2.
+
+The details of all changes can be found in
+[controller-runtime](https://github.com/kubernetes-sigs/controller-runtime/releases),
+[controller-tools](https://github.com/kubernetes-sigs/controller-tools/releases)
+and [kubebuilder](https://github.com/kubernetes-sigs/kubebuilder/releases)
+release notes.
+
+## Common changes
+
+- All three projects now use `go modules` instead of `dep`.
```

Comment:
```suggestion
- All three projects now use `go modules` instead of `dep`.  Dep support in `controller-runtime` will be maintained until the release of Go 1.13 (when modules are enabled by default).
```